param/__init__.py

```diff
@@ -1771,8 +1817,8 @@ class Path(Parameter):
 
     __slots__ = ['search_paths']
 
-    def __init__(self, default=None, search_paths=None, **params):
-        if search_paths is None:
+    def __init__(self, default=Undefined, search_paths=Undefined, **params):
+        if search_paths is Undefined:
             search_paths = []
```

Comment:
Mostly because the tests fail using `_slot_defaults` and I remembered the behavior of this Parameter is questionable and I have to return to https://github.com/holoviz/param/pull/546. I suggest not blocking this PR because of that, I'll work on https://github.com/holoviz/param/pull/546 next.